python/CMakeLists.txt

```diff
@@ -21,50 +20,59 @@ if (${Python_VERSION} VERSION_LESS 3.0.0)
   return()
 endif()
 
+add_subdirectory(src)
```

Comment:
Until I sort out how we need to structure our package to build a "cudaq" module off of "_pycudaq", I'm leaving the implementation files in `src` so they remain isolated from other directory changes.